libclambcc/ClamBCPrepareGEPsForWriter/ClamBCPrepareGEPsForWriter.cpp

```diff
@@ -156,6 +156,7 @@ class ClamBCPrepareGEPsForWriter : public ModulePass
         return 0;
     }
 
+use the functions from Common/ClamBCUtilities.cpp;
```

Comment:
Accidental copy/paste?

Reply:
This file isn't built yet under the port, so I am leaving that so that I don't forget to update it when I port it to clang 16.  I am still not sure whether or not I will even need this anymore.